jupyter_scheduler/models.py

```diff
@@ -269,6 +269,10 @@ class ListJobDefinitionsResponse(BaseModel):
     next_token: Optional[str] = None
 
 
+class CreateJobFromDefinition(BaseModel):
```

Comment:
I think this is a good start to the things that users will want to customize for this usage case.